brotab/extension/chrome/manifest.json

```diff
@@ -1,15 +1,14 @@
 {
-  // Extension ID: knldjmfmopnpolahpmmgbagdohdnhkik
```

Comment:
had to remove this comment cause parcel can't seem to handle json files with comments